src/prommis/nanofiltration/tests/test_diafiltration_two_salt.py

```diff
@@ -36,6 +36,86 @@
 from prommis.nanofiltration.diafiltration_two_salt import TwoSaltDiafiltration
 
 
+@pytest.mark.unit
+def test_zero_chi_implementation():
+    m = ConcreteModel()
+    m.fs = FlowsheetBlock(dynamic=False)
+    m.fs.properties = SoluteParameter()
+
+    m.fs.unit = TwoSaltDiafiltration(
+        property_package=m.fs.properties,
+        NFEx=10,
+        NFEz=5,
+        charged_membrane=False,
+    )
+
+    assert value(m.fs.unit.membrane_fixed_charge) == 0
+    assert m.fs.unit.diffusion_params_lithium_lithium == [
+        -4.07e-06,
+        -3.99e-09,
+        4.01e-09,
+    ]
+    assert m.fs.unit.diffusion_params_lithium_cobalt == [-9.62e-07, -1.03e-08, 1.04e-08]
+    assert m.fs.unit.diffusion_param_cobalt_lithium == [-5.24e-07, 2.49e-09, -2.50e-09]
+    assert m.fs.unit.diffusion_params_cobalt_cobalt == [-4.00e-06, 6.45e-09, -6.48e-09]
+    assert m.fs.unit.convection_params_lithium == [1, 0, 0]
+    assert m.fs.unit.convection_params_cobalt == [1, 0, 0]
```

Comment:
Instead of just being lists that live on the unit model, these should probably be indexed `Vars` because I'm sure we'll want to do parameter estimation sometime in the future.